server/config/routes.js

```diff
@@ -83,7 +83,7 @@ exports.configure = function (app) {
   winston.debug('Configuring routes');
 
   // exposed to the outside without authentication
-  app.get('/languages', users.getLanguages);
+  app.get('/languages', users.languages);
```

Comment:
GET is the verb ...  we discussed this as a team.  It is the HTTP verb, and it is getting languages.